clarity_epp.py

```diff
@@ -263,10 +270,20 @@ def placement_complete_step(args):
     parser_export_ped.set_defaults(func=export_ped_file)
 
     parser_export_removed_samples = subparser_export.add_parser(
-        'removed_samples', help='Export removed sampels table.', parents=[output_parser]
+        'removed_samples', help='Export removed samples table.', parents=[output_parser]
     )
     parser_export_removed_samples.set_defaults(func=export_removed_samples)
 
+    parser_export_sample_indications = subparser_export.add_parser(
+        'sample_indications', help='Export sample indication table.', parents=[output_parser]
+    )
+    parser_export_sample_indications.add_argument('-a', '--artifact_name',  nargs='?', help='Artifact name')
+    parser_export_sample_indications.add_argument('-r', '--sequencing_run',  nargs='?', help='Sequencing run name')
+    parser_export_sample_indications.add_argument(
+        '-p', '--sequencing_run_project',  nargs='?', help='Sequencing run project name'
+    )
```

Comment:
Are these arguments evaluated ? 
What would happen if I run the export and only use `-p` ?
`python clarity_epp.py export sample_indications -p CREv2_1`

What would happen if I run the export without any additional parameters?
`python clarity_epp.py export sample_indications `

I suspect in case of these scenarios, the script will execute [L151](https://github.com/UMCUGenetics/clarity_epp/blob/8c155092070bb26f27e6958febc8db7e9f90363e/clarity_epp/export/sample.py#L151)
However, it is slightly misleading... it would never find any samples, because user used wrong combination of parameters or none at all.

Reply:
`python clarity_epp.py export sample_indications -p CREv2_1` -> returns `Can't find sample(s).`
`python clarity_epp.py export sample_indications` -> returns `Can't find sample(s).`
So indeed both will result in L151.

I have fixed this in: 638a0eba27ebbe2f59715cd94f1448eaf4b7a557

```bash
python clarity_epp.py export sample_indications
usage: clarity_epp.py export sample_indications [-h] [-o [OUTPUT_FILE]] (-a ARTIFACT_NAME | -r SEQUENCING_RUN) [-p [SEQUENCING_RUN_PROJECT]]
clarity_epp.py export sample_indications: error: one of the arguments -a/--artifact_name -r/--sequencing_run is required

python clarity_epp.py export sample_indications -r 210716_A00295_0499_AH3FFJDMXY -a U209365CF2021D09279
usage: clarity_epp.py export sample_indications [-h] [-o [OUTPUT_FILE]] (-a ARTIFACT_NAME | -r SEQUENCING_RUN) [-p [SEQUENCING_RUN_PROJECT]]
clarity_epp.py export sample_indications: error: argument -a/--artifact_name: not allowed with argument -r/--sequencing_run

```